scenarios/27_Orannon.cfg

```diff
@@ -595,7 +723,13 @@
         {MSGa_Gawen _"Remember the Freetown! Remember the Freetown! Kill them all!"}
         [objectives]
             side=1
-            note=_"In this scenario, only Gawen and Lorin may interrogate enemy leaders"
+#ifdef NIGHTMARE
+            #po: no hints for NIGHTMARE difficulty:
+            note=_"You will have to figure out who has interrogations in this scenario on your own."
```

Comment:
I feel like you can just omit the note.

Reply:
omitted in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).